custom-pages/login/src/services/AuthApi.js

```diff
@@ -29,22 +29,22 @@ export default class AuthApi {
     this.instance = new auth0.WebAuth(params)
   }
 
-  async login(username, password) {
-    return this.instance.login(
+  login(username, password, cb) {
```

Comment:
can cb be "callback" please? just so it's clear.

Reply:
sure, do you mind if I do in the other PR?

Reply:
Sure